src/app/workspaces/create-workspace/ready-to-go-stacks/devfile-selector/devfile-selector.html

```diff
@@ -45,9 +45,9 @@
                     <div class="devfile-selector-item-icon">
                       <img img-src="{{devfile.icon}}"
                         img-src-onload="devfileSelectorController.iconOnLoad(devfile.icon, $result)"
-                        ng-show="devfileSelectorController.showIcon(devfile.icon)===true">
+                        ng-show="devfile.icon && devfileSelectorController.showIcon(devfile.icon)===true">
                       <div class="chefont cheico-type-blank"
-                        ng-show="devfileSelectorController.showIcon(devfile.icon)===false"></div>
+                        ng-show="!!devfile.icon===false || devfileSelectorController.showIcon(devfile.icon)===false"></div>
```

Comment:
I think the better variant to create a needed blob for catch method:
https://github.com/eclipse/che-dashboard/blob/98d3769ab123636dbf935a8e47eb7beecde9d228/src/components/attribute/img-src/img-src.directive.ts#L63
WDYT?

Reply:
answered here https://github.com/eclipse/che-dashboard/pull/5/files#r397661689